pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
+
+### Network
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Network.html)
 
-In IIDM, the network is constituted of [substations](#substation), which are themselves constituted of [voltage levels](#voltage-level).
-All the equipments are then connected to the voltage levels.
-The network comprises metadata in IIDM: 
-- a case date: the date and time of the target network that is being modelled
-- a forecast distance: the number of minutes between the network generation date and the case date
+In the PowSyBl grid model, the Network contains [substations](#substation), which themselves contain [voltage levels](#voltage-level).
 
-**Available extensions**
+| Attribute | Description |
+| --------- | ----------- |
+| $$SourceFormat$$ | Source format of the imported network model |
+| $$CaseDate$$ | Date and time of the target network that is being modeled |
+| $$ForecastDistance$$ | Number of minutes between the network generation date and the case date |
 
-- [CGMES conversion context extension]()
-- [CGMES model extension]()
+Only the `SourceFormat` attribute is required.
 
 ### Substation
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Substation.html)
 
-A substation in IIDM represents a specific geographical location with a set of equipments connected to one or several [voltage levels](#voltage-level).
-It comprises metadata in IIDM:
-- a country: to specify in which country the substation is located. It is an optional attribute, not set on fictitious test networks for example.
-- a set of geographical tags: they make it possible to accurately locate the substation
-- a [TSO](/pages/glossary.md#tso) information: to track to which TSO the substation belongs
+A substation represents a specific geographical location with equipment grouped in one or several [voltage levels](#voltage-level).
 
-**Available extensions**
+| Attribute | Description |
+| --------- | ----------- |
+| $$Country$$ | To specify in which country the substation is located |
+| $$GeographicalTags$$ | They make it possible to accurately locate the substation |
+| $$TSO$$ | To track to which [TSO](/pages/glossary.md#tso) the substation belongs |
 
-- [ENTSOE Area]()
+All three attributes are optional.
 
 ### Voltage Level
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/VoltageLevel.html)
 
-A voltage level in IIDM represents a set of equipments connected together with the same nominal voltage, physically close to each other (~ 1-100m).
-Two voltage levels may be connected through a line (they are then located in different substations) or through transformers (they are then located within
-the same substation).
+A voltage level contains equipment with the same nominal voltage.
```

Comment:
equipments?

Reply:
I think that equipment does not have a plural form.

Reply:
@marqueslanauja: I'm not sure that the mistake has not been done elsewhere. Could you check (and create a new PR to fix "equipments" mistake)

Reply:
https://github.com/powsybl/powsybl.github.io/issues/183